packages/web/src/types.ts

```diff
@@ -17,11 +17,22 @@ export interface AnalyticsProps {
   beforeSend?: BeforeSend;
   debug?: boolean;
   mode?: Mode;
+  route?: string | null;
+
+  disableAutoTrack?: boolean;
+
+  scriptSrc?: string;
+  endpoint?: string;
+
+  dsn?: string;
```

Comment:
**question:** why not documenting `framework` here as well?

Reply:
we also didn't do this in Speed Insights. I guess we should not allow them to send a custom one so we have cleaner data